client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -6,7 +6,7 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
 import { includes, isEmpty, noop, flowRight, has, trim, sortBy, reverse } from 'lodash';
-import url from 'url';
+import url from 'url'; // eslint-disable-line no-restricted-imports
```

Comment:
@yoavf just wanted to give you all a heads up that this library is being deprecated. I tried to clean it up in this PR but it ended up being a little more involved than I wanted for this PR.